src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -0,0 +1,89 @@
+  ~     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+  ~     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+  ~     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  ~     POSSIBILITY OF SUCH DAMAGE.
+  -->
+
+<template>
+    <div class="slds-m-bottom_medium">
+        <c-util-alert-banner theme="success"
+                             icon="utility:success"
+                             size="small"
+                             message="Gateway updated."
+                             is-dismissable="true"
+                             is-visible={isSuccess}>
+        </c-util-alert-banner>
+    </div>
+    <template if:true={isError}>
+        <div class="slds-m-bottom_medium">
+            <c-util-alert-banner theme="error"
+                                 icon="utility:error"
+                                 size="small"
+                                 message={errorMessage}
+                                 is-dismissable="true"
+                                 is-visible={isError}>
+            </c-util-alert-banner>
+        </div>
+    </template>
+    <lightning-layout multiple-rows="true">
+        <div if:true={showSpinner}
+             class='spinner-container'>
+            <lightning-spinner alternative-text={CUSTOM_LABELS.messageLoading}
+                               size='large'>
+            </lightning-spinner>
+            <p class='loading-text slds-text-heading_medium'>{loadingText}</p>
+        </div>
+        <lightning-layout-item size="12">
+            <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
+        </lightning-layout-item>
+        <lightning-layout-item size="12">
+            <p class="slds-text-body_regular slds-p-vertical_medium">NPSP uses the gateway that was added to Elevate most
+                recently. To override this, click Edit and enter the gateway ID for the one you'd like to use instead.
+                Find the gateway ID in the Elevate Admin Console | Gateway Management.</p>
+        </lightning-layout-item>
+        <template if:true={isReadOnly}>
+            <lightning-layout-item size="12" class="slds-text-align_center">
+                <lightning-button label="Edit" data-id="editGatewayButton" onclick={handleEdit}></lightning-button>
+            </lightning-layout-item>
+        </template>
+        <lightning-layout-item size="4">
+            <lightning-input variant="label-inline"
+                             read-only={isReadOnly}
+                             label="Gateway ID"
+                             value={gatewayId}
+                             type="text"
+                             data-id="gatewayIdField">
+            </lightning-input>
+            <template if:false={isReadOnly}>
+                <lightning-button-group>
+                    <lightning-button label="Save" variant="brand" onclick={handleSave}></lightning-button>
```

Comment:
Can we use custom labels for these button labels?

Reply:
I didn't see any custom labels we could use that had common enough names and since we aren't using any new labels for the most part I just hard coded